Antonio_Ramos_Documentation

```diff
@@ -0,0 +1,171 @@
+
+- Browsed code and familiarized myself with the file organizational structure.
+
+- Identified location of all installed modules that are being imported by python scipt 
+ "G:\MAX-Filer\Research\Labs-kbuzard-S18\kfunctions\Three Stage K-functions\K-function Local_3Stage Trial" they are located at "C:\Python4Arc\ArcGIS\Lib\site-packages"
+
+- packages  “geopandas” and "shapely" missing from "C:\Python4Arc\ArcGIS\Lib\site-packages"
+
+- Decided to attempt to install packages "shapely" and "geopandas" to C:\Apps-SU\Anaconda3 so that we can test run the python script Local_3Stage.py and see if it will get past the module importation. 
+
+— Began Attempt to install package “shapely”. Opened Anaconda command prompt and entered “conda install shapely”. 
+
+
+- package "shapely" successfully installed to "C:\App-SU\Anaconda3".
+
+- when attempting to instal geopandas with "conda install geopandas"  an inconsistent environment error arised.
+
+- after reviewing stackoverflow I followed the suggestion of updating all modules with command "conda update --all".
+
+- the update was interupted due to a lack of administrator privaledge on my user. This corrupeted file "C:\Apps-SU\Anaconda3" and I had to work with ICT to resolve the issue and disentagle the conflicting dependancies the corrupted file was renamed "C:\Apps-SU\Anaconda3-old" and ICT created new file "C:\Apps-SU\Anaconda3" with reinstall of Anaconda and all packages that had been installed to "C:\Apps-SU\Anaconda3-old" this includes the shapely package installed earlier but not geopandas 
+
+- after working with ICT spyder4 was not opening, so the files located at "C:\Users\YOUR_USER_NAME\.spyder-py3\"   " C:\Users\YOUR_USER_NAME\.ipython" "C:\Users\YOUR_USER_NAME\.matplotlib"   were all deleted with copies placed in "C:\Users\aramosri\documents"
+
+- spyder 4 now opens correctly
+
+- upon opening spyder i was asked if I wanted to install "Kite" which "provides better and more accurate code completion in its editor when using packages like matplotlib numpy and pandas (Which are all being used in our python scripts) -- will ask prof buzard if she believes we should install
+
```

Comment:
It's good practice to go back and update when things like this are resolved (e.g. we decided not to install Kite)

It's also helpful to divide this up by date--it often helps to make sense of things in retrospect

Reply:
Ill go back to add that note, and I think I can still divide it up by date, for the most part I remember what I did on the same days. S o while the particular dates may be off by one or two days, the grouping of things I did each day will not be.